src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static String SUPPLY = "supply";
+    private static String BUY = "buy";
+    private static String RESULT = "result";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String dataFromFile = readLinesFromFile(fromFileName);
+        String report = makeReport(dataFromFile);
+        writeToFile(toFileName, report);
+    }
+
+    private String readLinesFromFile(String fromFileName) {
+        File fileFrom = new File(fromFileName);
+        StringBuilder stringBuilder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileFrom))) {
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                stringBuilder.append(value).append(" ");
+                value = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read the file " + fromFileName, e);
+        }
+        return stringBuilder.toString();
+    }
+
+    private String makeReport(String linesFromFile) {
+        int supply = 0;
+        int buy = 0;
+        String[] oneString = new String[2];
+        String[] linesArray = linesFromFile.split(" ");
+        for (int i = 0; i < linesArray.length; i++) {
+            oneString = linesArray[i].split(",");
+            if (oneString[0].equals(SUPPLY)) {
+                supply += Integer.parseInt(oneString[1]);
+            } else {
+                buy += Integer.parseInt(oneString[1]);
+            }
+        }
```

Comment:
```suggestion
        String[] linesArray = linesFromFile.split(" ");
        for (int i = 0; i < linesArray.length; i++) {
            String[] operations = linesArray[i].split(",");
            if (operations[0].equals(SUPPLY)) {
                supply += Integer.parseInt(operations[1]);
            } else {
                buy += Integer.parseInt(operations[1]);
            }
        }
```
Also create constants for array indexes